docs/app-areas.md

```diff
@@ -1,37 +1,28 @@
-An AEA is an intelligent agent whose goal is generating economic value for its owner. It can represent machines, humans (individuals or organisations), or data. 
+The <a href="../agent-oriented-development">agent-oriented development</a> guide presented some of the fundamental attributes of the environments which AEA-based solutions are suited for.
 
-## General application areas
+Under the light of that discussion, in this page we identify a number of application areas for AEA-based solutions. This list is by no means comprehensive. In fact, we are most excited about applications we have not thought of before.
 
-As described in the guide on <a href="../agent-oriented-development">agent-oriented development</a>, AEAs are designed to operate in an environment with:
-
-* multiple stakeholders, which
-* are represented by AEAs, that
-* interact autonomously and
-* communicate _via_ a peer-to-peer network.
```

Comment:
Please keep the repetition. It's ok to repeat content on other pages as long as it fits. Here this is helpful as it summarises the environment characteristics.

Reply:
Done